src/lib/components/edit-transaction.svelte

```diff
@@ -51,7 +52,16 @@
 	let endDate = $state(new Date())
 	let period = $state(30)
 	let periodUnit: TimeUnit = $state('year')
-	const numOccurences = $derived(dateDifferenceFunction(repeatUnit)(endDate, date))
+	const numOccurences = $derived(
```

Comment:
Please also fix the typo in the variable name:

```suggestion
	const numOccurrences = $derived(
```